### DISCUSSION TITLE
ユーザごとのハイライトの仕様について質問

### DISCUSSION THREAD

**Discussion by anatawa12** [misskey-dev/misskey#12473] [Q&A] [answered]
現在ユーザ毎のハイライトについてはハイライトのスコア順ではなく時系列順になっていますがこれは意図した仕様でしょうか

ハイライトという名称から、完全な時系列順は違和感を感じました。

**Comment by syuilo** [comment, marked as answer]
普通のハイライトも時系列順ですね
どちらも意図した実装です

**Reply by anatawa12** [reply]
ではハイライト用のスコアのランキングはどのような用途で使われてる感じでしょうか？ハイライトに入ってるか否か以上の情報がないように思ったのですが
